spyder/config/base.py

```diff
@@ -228,6 +228,39 @@ def get_conf_path(filename=None):
         return osp.join(conf_dir, filename)
 
 
+def get_conf_paths():
+    """Return the files that can update system configuration defaults."""
+    CONDA_PREFIX = os.environ.get('CONDA_PREFIX', None)
+
+    if os.name == 'nt':
+        SEARCH_PATH = (
+            'C:/ProgramData/spyder',
+        )
+    else:
+        SEARCH_PATH = (
+            '/etc/spyder',
+            '/usr/local/etc/spyder',

```

Comment:
Are these paths the right ones in macOS? What's the output of `jupyter --paths` there?

Reply:
```bash
$ jupyter --paths
config:
    /Users/gpena-castellanos/.jupyter
    /Users/gpena-castellanos/anaconda2/envs/spy37/etc/jupyter
    /usr/local/etc/jupyter
    /etc/jupyter
data:
    /Users/gpena-castellanos/Library/Jupyter
    /Users/gpena-castellanos/anaconda2/envs/spy37/share/jupyter
    /usr/local/share/jupyter
    /usr/share/jupyter
runtime:
    /Users/gpena-castellanos/Library/Jupyter/runtime
```

Reply:
Thanks for the confirmation @goanpeca!